specificatie/kadastraal-onroerende-zaken.yaml

```diff
@@ -188,8 +188,6 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/404"
         '406':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/406"
-        '410':
```

Comment:
Is de situatie rondom vervallen appartementsrechten zoals beschreven in #825 veranderd ? Daar werd de conclusie getrokken dat de 410 nog wel nodig is.

Reply:
Appartementen die zijn vervallen (& ook kadastrale objecten die zijn vervallen bij een wettelijke herverkaveling) geven een "niet gevonden" foutmelding in de bron. Dit lijkt te komen doordat deze situaties geen overgegaan in kennen.